docs/en/observability/troubleshoot-uptime.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[troubleshoot-uptime]]
+= Uptime troubleshooting
+
+== Mapping issues
+
+It is possible for Heartbeat to create an index without the appropriate mappings.
+When this happens, the {uptime-app} will be unable to get the data it requires; in this
+state, the UI cannot function.
+
+To fix this problem, you typically need to remove your Heartbeat indices, and create
+new ones, with the appropriate mappings installed. To achieve this, follow the steps below.
+
+=== Step 1: Delete your Heartbeat indices
+
+["source","sh"]
+----
+curl -u {USERNAME}:{PASSWORD} -X DELETE {INDEX_URL}
```

Comment:
I was unsure what was appropriate to use in this case; my thinking was by including these missing values the user would be less likely to simply copy/paste and delete something before they were ready.

Reply:
I would avoid cURL here and make the advice more generic. People can do this in a number of ways, including the kibana management UI. I don't think we need a code example at all. I'd maybe link to the kibana mgmt docs and the ES index delete api docs.

Reply:
Removed the comment, linked to Index Management and Delete Index API docs https://github.com/elastic/observability-docs/pull/1046/commits/5e439ac02744484eff67ea9612e817cb09c02f71.